lib/Models/CatalogItem.js

```diff
@@ -764,6 +764,17 @@ var scratchRectangle = new Rectangle();
 };
 
 /**
+ * Returns true if this item currently has a rectangle to zoom to. Depends on observable properties, and so updates once loaded.
+ * @returns {Boolean} A flag for whether this catalog item supports "zoom to".
+ */
+CatalogItem.prototype.canZoomTo = function() {
+    if (defined(this.nowViewingCatalogItem)) {
+        return defined(this.nowViewingCatalogItem.rectangle);
```

Comment:
You should call `this.nowViewingCatalogItem.canZoomTo` instead of checking the `rectangle` property directly.  That way a derived catalog item can override the behavior.

Reply:
Also, should this bit a property on the prototype instead of a function?